content/en/docs/tasks/traffic-management/egress/egress-tls-origination/index.md

```diff
@@ -17,6 +17,11 @@ This example shows how to configure Istio to perform {{< gloss >}}TLS originatio
 for traffic to an external service. Istio will open HTTPS connections to the external service while the original
 traffic is HTTP.
 
+{{< tip >}}
+This is currently broken for servers that validate ALPN strings.
```

Comment:
What is the ETA for fixing this bug? If it will be fixed in Istio 1.9, then the warning isn't needed in the `master` branch, but instead should be sent to branch `release-1.8`. Or maybe it's going to be fixed in the next point release, and isn't needed at all? cc @howardjohn